backbone.js

```diff
@@ -1440,6 +1440,11 @@
       return path === this.root && !this.getSearch();
     },
 
+    // Decode unicode escapes without decoding `%25`.
```

Comment:
> Can you add a bit more prose here for future-usses who might look at this quizzically?

Happy to!

> Also, any reason this has to be on the Router prototype instead of living as a standalone helper function?

I started out with a standalone helper but then I started to think what would need to be done if someone finds another problematic escape sequence. With the helper, you'd need to override `#getPath` and `#navigate`, which could get ugly really fast. This way, it's a one liner to replace `#decodeFragment`.